src/main/webapp/components/Editor/Profile.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import './Profile.css'
+
+function Profile(props) {
+  const {name, side, active} = props;
+
+  return (
+    <div className={`Profile_container ${side==="R"? "Profile_flipped": ""}`}>
```

Comment:
Nit: add space before ? and : for the ternary operator.

`condition ? exprIfTrue : exprIfFalse`

Reply:
Okay thank you. I will fix it.